### ISSUE TITLE
Unclear doc comments for URL positional argument parsed as URL without a hostname value

### ISSUE THREAD

**Issue by atc0005** [atc0005/check-cert#384]
We attempt to parse a given server value (`serverVal`) here:

https://github.com/atc0005/check-cert/blob/9d84bf80f89582fdfe63221f84c404ad35fb1745/internal/config/parse.go#L31

This doc comment:

https://github.com/atc0005/check-cert/blob/9d84bf80f89582fdfe63221f84c404ad35fb1745/internal/config/parse.go#L82-L88

doesn't properly convey *why* we're using `serverVal` as-is when URL parsing doesn't result in a hostname value.

This earlier doc comment adds some additional context:

https://github.com/atc0005/check-cert/blob/9d84bf80f89582fdfe63221f84c404ad35fb1745/internal/config/parse.go#L45-L48

In short, if a parsed positional argument results in a missing `u.Host` value (but not a parsing error), it is missing the protocol scheme and is likely a hostname, FQDN or IP Address. We use the value as-is so that the application (`lscert` in our case) can later attempt to resolve the value (hostname or FQDN).
